kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/ConfigDrift.java

```diff
@@ -97,6 +101,9 @@ public Result check(ConfiguredTopic desired, ConfiguredTopic actual, Type type)
           }
         }
 
+      case REPLICATION_FACTOR:
+	return desired.getReplicationFactor() == actual.getReplicationFactor() ? Result.NO_DRIFT: Result.UNSUPPORTED_DRIFT;
```

Comment:
this has odd indentation. Is your IDE setup with correct code style?

Reply:
it wasn't at the time. I've got it setup with autoformat and two-space indentation now. I notice though that it seems my autoformatter is making some different decisions with regards to breaking up lines than in the rest of the code base. do you have a particular style configuration which you use?